index.js

```diff
@@ -482,7 +504,16 @@ export default class WebTorrent extends EventEmitter {
     if (this._connPool) {
       // Sometimes server.address() returns `null` in Docker.
       const address = this._connPool.tcpServer.address()
-      if (address) this.torrentPort = address.port
+      if (address) {
+        this.torrentPort = address.port
+        if (this.natTraversal != null) {
+          try {
+            this.natTraversal.map({ publicPort: this.torrentPort, privatePort: this.torrentPort, protocol: this.utp ? null : 'tcp', description: 'WebTorrent Torrent' })
```

Comment:
this is async, so yo need a .catch() not a try-catch unless you make this function async and use await

Reply:
Thanks for catching this, my head clearly wasn't on straight when writing this. This has been amended to .catch the error.